pkgs/development/libraries/qt-6/modules/qtbase.nix

```diff
@@ -222,8 +222,8 @@ stdenv.mkDerivation rec {
     "-DQT_FEATURE_journald=ON"
     "-DQT_FEATURE_vulkan=ON"
   ] ++ lib.optionals stdenv.isDarwin [
-    # build as a set of dynamic libraries
-    "-DFEATURE_framework=OFF"
```

Comment:
What's the benefits/implications of building as frameworks?

Reply:
qmake generates erroneous link args (e.g. `-lQtNetwork.1`) when not building as frameworks on darwin, so I don't have much choice...